src/current/molt/molt-replicator.md

```diff
@@ -0,0 +1,693 @@
+
+{% include_cached copy-clipboard.html %}
+~~~
+--sourcePDBConn $SOURCE_PDB
+~~~
+
+Specify the source Oracle schema to replicate from:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--sourceSchema migration_schema
+~~~
+</section>
+
+To replicate from the correct position, specify the appropriate checkpoint value. 
+
+<section class="filter-content" markdown="1" data-scope="postgres">
+Use `--slotName` to specify the slot created during the data load, which automatically tracks the LSN (Log Sequence Number) checkpoint:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--slotName molt_slot
+~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="mysql">
+Use `--defaultGTIDSet` from the `cdc_cursor` field in the MOLT Fetch output:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--defaultGTIDSet '4c658ae6-e8ad-11ef-8449-0242ac140006:1-29'
+~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="oracle">
+Use the `--scn` and `--backfillFromSCN` values from the MOLT Fetch output:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--scn 26685786
+--backfillFromSCN 26685444
+~~~
+</section>
+
+Specify the staging database. Use `--stagingCreateSchema` to create it automatically on first run:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--stagingSchema _replicator
+--stagingCreateSchema
+~~~
+
+At minimum, the `replicator` command should include the following flags:
+
+<section class="filter-content" markdown="1" data-scope="postgres">
+{% include_cached copy-clipboard.html %}
+~~~ shell
+replicator pglogical \
+--sourceConn $SOURCE \
+--targetConn $TARGET \
+--slotName molt_slot \
+--stagingSchema _replicator \
+--stagingCreateSchema
+~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="mysql">
+{% include_cached copy-clipboard.html %}
+~~~ shell
+replicator mylogical \
+--sourceConn $SOURCE \
+--targetConn $TARGET \
+--defaultGTIDSet '4c658ae6-e8ad-11ef-8449-0242ac140006:1-29' \
+--stagingSchema _replicator \
+--stagingCreateSchema
+~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="oracle">
+{% include_cached copy-clipboard.html %}
+~~~ shell
+replicator oraclelogminer \
+--sourceConn $SOURCE \
+--sourcePDBConn $SOURCE_PDB \
+--sourceSchema migration_schema \
+--targetConn $TARGET \
+--scn 26685786 \
+--backfillFromSCN 26685444 \
+--stagingSchema _replicator \
+--stagingCreateSchema
+~~~
+</section>
+
+For detailed steps, refer to [Load and replicate]({% link molt/migrate-load-replicate.md %}#start-replicator).
+
+### Resume after interruption
+
+<div class="filters filters-big clearfix">
+    <button class="filter-button" data-scope="postgres">PostgreSQL</button>
+    <button class="filter-button" data-scope="mysql">MySQL</button>
+    <button class="filter-button" data-scope="oracle">Oracle</button>
+</div>
+
+When resuming replication after an interruption, MOLT Replicator automatically uses the stored checkpoint to resume from the correct position. 
```

Comment:
Might be good to reiterate: it's important to reuse the same `--stagingSchema` you did on previous runs of Replicator